### ISSUE TITLE
Lock files are not created or removed

### ISSUE THREAD

**Issue by TonioF** [bcdev/zappend#55]
I tried to run zappend (v 0.4.0) from the command line with 

```
zappend -t out/output.zarr in/*.zarr
```

Computation failed with message: 
```
Error: [Errno 2] No such file or directory: '.../out/output.zarr.lock'
```

After I created the .lock file myself, computation worked. After it finished successfully, I had to remove the file which was still there. My processing system is CentOS.

As I write this down, I notice that the lock file stated above is not correct, as it should be within the zarr folder, which it isn't.

**Comment by forman** [comment]
@TonioF thanks, this is really  strange, never seen this problem so far.
Could you please try the same using the Python `zappend()` function, so we get a traceback. 
Btw, a traceback also when using the CLI would be good to have.

**Comment by forman** [comment]
traceback: #57

**Comment by forman** [comment]
The issue was a misleading error message. The root cause is that the target parent directory must exist, `out` in your case obviously did not.

Closed by #63
